docs/reference/pc_renaming.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Package Control: Renaming a package"
+---
+
+<!-- Originals: -->
+<!-- https://packagecontrol.io/docs/renaming_a_package -->
+<!-- https://github.com/wbond/packagecontrol.io/blob/master/app/html/docs/renaming_a_package.html -->
+
+
+# Renaming a package
+
+Let's guide you through the process of renaming a package in the [default Package Control channel][pcc].
+
+The goal is to update the main channel so that:
+
+- your entry gets its new `name`
+- is sorted alphabetically
+- has its previous name in the array of `previous_names`
```

Comment:
```suggestion
The goal is to update the main channel so that your entry:

- gets its new `name`
- is sorted alphabetically
- has its previous name in the array of `previous_names`
```

or

```suggestion
The goal is to update the main channel so that:

- your entry gets its new `name`
- your entry is sorted alphabetically
- your entry has its previous name in the array of `previous_names`
```